src/core/AutomationPattern.cpp

```diff
@@ -82,8 +85,8 @@ AutomationPattern::AutomationPattern( const AutomationPattern & _pat_to_copy ) :
 	for( timeMap::const_iterator it = _pat_to_copy.m_timeMap.begin();
 				it != _pat_to_copy.m_timeMap.end(); ++it )
 	{
-		m_timeMap[it.key()] = it.value();
-		m_tangents[it.key()] = _pat_to_copy.m_tangents[it.key()];
+		// Copies the automation node (in/out values and in/out tangents)
+		m_timeMap[POS(it)] = it.value();
```

Comment:
Let's add a note about this in the code itself as a heads up/reminder.

Reply:
Sure, where should I add it though? I think maybe adding it to the `AutomationNode` header would be better? Because when adding a dynamic allocated member to the `AutomationNode` class the developer might not look at this line.

Reply:
That's the hard part to figure out. It wouldn't hurt to add it to multiple spots though.